src/main/java/koreatech/in/domain/NotiSlack.java

```diff
@@ -51,9 +52,12 @@ public static NotiSlack deleteCompleteNotiSlack(User user) {
     }
 
     public static NotiSlack ownerShopRequestNotiSlack(User user) {
+        String newLine = System.lineSeparator();
+        String appendedTextAdminPageUrl = OWNERSHOP_REQUEST_SUFFIX + newLine + MARKDOWN_ADMIN_PAGE_URL;
```

Comment:
이거는 한줄로해도 좋을듯합니다.
가독성 조금 더 신경쓰고싶으면 이것도 고려해볼수 있을거같네요

어디까지나 취향차이라서 아래 한수가 이야기한것처럼 크게 신경쓸 부분은 아니라 저도 Approve 해두겠습니다.
```suggestion
        String ownerMessageWithPageUrl = String.format("%s %s %s",
                OWNERSHOP_REQUEST_SUFFIX,
                newLine,
                MARKDOWN_ADMIN_PAGE_URL
        );
```